database/migrations/2016_12_27_083228_alter_product_price_value.php

```diff
@@ -28,7 +28,7 @@ public function down()
     {
         //
         Schema::table('products', function (Blueprint $table) {
-            $table->dropColumn('price', 8, 2)->change();
+            $table->decimal('price', 8, 2)->change();
```

Comment:
@madindo if you wanna make change to exsisting field in tables, create new migration file to alter the table